apps/web/cypress/tests/integration-store.spec.ts

```diff
@@ -5,46 +5,84 @@ describe('Integration store page', function () {
 
-    getFirstIntegrationCard().get('button').contains('Connect');
-  });
+      getFirstIntegrationCard().get('button').contains('Connect');
+    });
 
-  it('should display integrated sendgrid provider', function () {
-    interceptIntegration(true);
+    it('should display integrated sendgrid provider', function () {
+      interceptSendgridIntegration(true);
 
-    cy.visit('/integrations');
+      cy.visit('/integrations');
 
-    getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Active');
-  });
+      getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Active');
+    });
+
+    it('should display not integrated sendgrid provider', function () {
+      interceptSendgridIntegration(false);
+
+      cy.visit('/integrations');
+
+      getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Not Active');
+    });
+
+    it('should display use credentials on settings modal', function () {
+      interceptSendgridIntegration(true);
 
-  it('should display not integrated sendgrid provider', function () {
-    interceptIntegration(false);
+      cy.visit('/integrations');
 
-    cy.visit('/integrations');
+      getFirstIntegrationCard().getByTestId('provider-card-settings-svg').click();
 
-    getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Not Active');
+      cy.getByTestId('apiKey').should('have.value', '123');
+      cy.getByTestId('from').should('have.value', 'cypress');
+    });
   });
 
-  it('should display use credentials on settings modal', function () {
-    interceptIntegration(true);
+  describe('Nodemailer', () => {
+    it('should display email available for connection', function () {
+      cy.visit('/integrations');
+      cy.location('pathname').should('equal', '/integrations');
+
+      getFirstIntegrationCard().get('button').contains('Connect');
+    });
+
+    it('should display integrated nodemailer provider', function () {
+      interceptNodemailerIntegration(true);
+
+      cy.visit('/integrations');
+
+      getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Active');
+    });
+
+    it('should display not integrated sendgrid provider', function () {
+      interceptNodemailerIntegration(false);
+
+      cy.visit('/integrations');
 
-    cy.visit('/integrations');
+      getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Not Active');
+    });
 
-    getFirstIntegrationCard().getByTestId('provider-card-settings-svg').click();
+    it('should display use credentials on settings modal', function () {
+      interceptNodemailerIntegration(true);
 
-    cy.getByTestId('apiKey').should('have.value', '123');
-    cy.getByTestId('from').should('have.value', 'cypress');
+      cy.visit('/integrations');
+
+      getFirstIntegrationCard().getByTestId('provider-card-settings-svg').click();
+
+      cy.getByTestId('apiKey').should('have.value', '123');
```

Comment:
@p-fernandez seems that nodemailer doesn't have apikey in provider-credentials

Reply:
Thanks for spotting it. Fixed in a different PR.